README.md

```diff
@@ -1,44 +1,35 @@
-# MeiliSearch Documentation
+# MeiliSearch documentation
 
-MeiliSearch is a **RESTful search API** that is the **ready-to-go solution** for everyone wanting a **powerful, fast, and relevant search experience** for their end-users ⚡️🔎
+This repository contains the documentation for [MeiliSearch](https://github.com/meilisearch/meilisearch), an open-source search engine API.
 
-[<linkButton text="🚀 GETTING STARTED"/>](/guides/introduction/quick_start_guide.md)
+## Contributing
 
-Efficient search engines are often only accessible to companies with the financial means and resources necessary to develop a search solution adapted to their needs. The majority of other companies that do not have the means or do not realize that the lack of relevance of a search greatly impacts the pleasure of navigation on their application,
-end up with poor solutions that are more frustrating than effective, for both the developer and the user.
+Want to contribute to the documentation of MeiliSearch? You can do so by cloning this repository, making your change and submitting your pull request.
 
-That's why we created MeiliSearch, an open-source solution accessible to everyone, meeting the vast majority of needs, even specific ones. Installable very easily with little or no configuration required but with a high capacity for customization.
+### Install & Run
 
-Our solution is **instant**; it **accepts typos**; it understands **filters**, **custom rankings**, and a lot of other [features](/#features).
+```bash
+# Clone the repository
+$ git clone git@github.com:meilisearch/documentation.git meilisearch-documentation && cd meilisearch-documentation
+# Install dependencies
+$ yarn install
+# Run MeiliSearch documentation on https://localhost:8080
+$ yarn dev
+```
 
-## Open-source
+#### Check dead links
 
-MeiliSearch is open-source. You can **support the project by starring** it on [our GitHub](https://github.com/meilisearch/MeiliSearch)!
+It can be tough to make changes to the documentation without creating any dead links. You can check the dead links before making any pull request
+```bash
+$ yarn check-links
+```
 
-<a class="github-button" href="https://github.com/meilisearch/MeiliSearch" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star meilisearch/MeiliSearch on GitHub">Star</a>
-<a class="github-button" href="https://github.com/meilisearch/MeiliSearch/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="false" aria-label="Fork meilisearch/MeiliSearch on GitHub">Fork</a>
-<script async defer src="https://buttons.github.io/buttons.js"></script>
+### Deployment
 
-## Demo
+The documentation is deployed as a static website.
 
-![crates.io demo gif](/crates-io-demo.gif)
-> Meili helps the Rust community find crates on [crates.meilisearch.com](https://crates.meilisearch.com)
+```bash
+$ yarn build
+# The website is now available in .vuepress/dist and you can serve it using any webserver.
+```
```

Comment:
Perhaps it's preferable to explain about the fact that we are using auto-deploy on master instead of telling the user he can deploy his own MeiliSearch documentation on his server.